extensions/oidc/deployment/src/main/resources/dev-ui/qwc-oidc-provider.js

```diff
@@ -1098,6 +1118,38 @@ export class QwcOidcProvider extends QwcHotReloadElement {
         }
     }
 
+    static _checkSessionCookie(jsonRpc, onLoggedIn, onLoggedOut) {
+        // FIXME: port, path?
+        fetch("http://localhost:8080/q/io.quarkus.quarkus-oidc/readSessionCookie")
```

Comment:
Sorry for lazy question as I didn't run your code yet, but I thought `fetch` doesn't send `httpOnly` cookies as it can't access them. I thought OIDC cookies are HTTP only. You tried this and it works?

Reply:
Sorry, it will work on same origin. I don't have rights to close this comment thread, so please resolve it. Thanks

Reply:
Actually, after reading https://developer.mozilla.org/en-US/docs/Web/API/fetch I'm still not sure it can work for `httpOnly`. If you tried it please resolve this thread.

Reply:
Actually, `httpOnly` means cookies can _only_ be sent to the server and not read by JavaScript code, which is precisely what I'm doing here: I have an endpoint whose sole purpose is to read the cookie sent by the browser to return it back to the JS caller. So this works, and it's the only way to make it work, unfortunately.

Reply:
Alright. Thanks for the explanation.